testsuite/classlibrary/TestWebView.sc

```diff
@@ -0,0 +1,51 @@
+TestWebView : UnitTest {
+
+	test_findText_found {
+		var window = Window();
+		var webView = WebView(window, Rect(0, 0, 100, 100));
+		var cond = Condition.new;
+		var wasFound, timeout;
+
+		window.front; // FIXME findText doesn't work if the window isn't displayed
+		webView.setHtml("abc");
+		0.1.wait;
+		webView.findText("abc", true, { |found|
+			"found".postln;
+			cond.unhang;
+			wasFound = found;
+		});
+		timeout = fork {
+			0.1.wait;
+			cond.unhang;
+		};
+		
```

Comment:
When I saved this file in scide I think this whitespace was automatically deleted... also in line 27.